### ISSUE TITLE
Problem of Tagger 1.8 with iTunes

### ISSUE THREAD

**Issue by fsaligny** [Bilalh/Tagger#60]
Tagger Version 1.8.0.7 (566) don't transfer the tag " grouping " into iTunes new version 12.5.3.16.

**Comment by Bilalh** [comment]
Did it work use to work? 
What version of OSX/macOS are you using?

**Comment by fsaligny** [comment]
I use Mac Sierra 10.12.1 on
iMac 21,5 pouces, fin 2013 processor 3,1 GHz Intel Core i7 memory 16 Go 1600 MHz DDR3
but no problems with Mac Sierra before new version of iTunes.

**Comment by Bilalh** [comment]
I still using 10.11, but I try to find the problem .

**Comment by fsaligny** [comment]
OK, do you have any information about my problem ?

**Comment by Bilalh** [comment]
Are you tagging mp3s?  I only get the bug to appear when using mp3s,  m4a/aac work correctly for me. 

The problem is that the current version of iTunes  is not reading or writing the grouping field (TIT1) of the id3v2 tag of the mp3.   

If you edit the grouping field (e.g. to TEST)  in iTunes then remove the track from iTunes and re-add it, the change made to grouping field  will not be there.

**Comment by fsaligny** [comment]
Yes, I use only mp3 format. If I have understood, it is impossible to solve the problem ?

**Comment by Bilalh** [comment]
Unlikely, unless apple fixes it. I will report the issue to apple to see if they fix their reading of the grouping field.

**Comment by fsaligny** [comment]
OK: I tried 'Tag Editor' (Apple store) and the same problem appears ! 
But thank you for assistance.

**Comment by ijabz** [comment]
Some helpful advice from a competitor (songkong), the problem is that iTunes have now mapped the TIT1 frame to the Work field for their classical music improvements, and have created a new frame GRP1 to use for their Grouping field http://blog.jthink.net/2016/12/itunes-add-new-grouping-field-but.html  ,the problem is for any format that supports ID3 (i.e Mp3, Aiff)

**Comment by Bilalh** [comment]
Thanks.  I am not sure what the best way of displaying this to the user would be.  I could call `GRP1` `iTunes grouping`. But for `TIT1` calling it `Work` would be confusing to everyone not using iTunes and calling it `grouping` would be confusing to iTunes users.  I test out some ideas at some point.

**Comment by ijabz** [comment]
Yes, it is problematic, much better if they had continued to use TIT1 for grouping and created a new frame for WORK (which is essentially what they did for the mp4 format)
